frontend/pages/hosts/details/HostDetailsPage/HostDetailsPage.tsx

```diff
@@ -461,12 +463,25 @@ const HostDetailsPage = ({
     data: hostCertificates,
     isLoading: isLoadingHostCertificates,
     isError: isErrorHostCertificates,
-  } = useQuery(
-    ["hostCertificates", host_id],
-    () => hostAPI.getHostCertificates(hostIdFromURL),
+  } = useQuery<
+    IGetHostCertificatesResponse,
+    Error,
+    IGetHostCertificatesResponse
+  >(
+    ["host-certificates", host_id],
```

Comment:
Consider adding pagination and sort params to the query key now as future-proofing

Reply:
updated.